TLM/SharedAssemblyInfo.cs

```diff
@@ -20,4 +20,4 @@
 //      Minor Version
 //      Build Number
 //      Revision
-[assembly: AssemblyVersion("11.6.0.*")]
+[assembly: AssemblyVersion("11.7.0.*")]
```

Comment:
is it really for 11.7? I didn't release 11.6 yet.

Reply:
yes because its a major API change that is required by AR.

depends on how we version stuff. I think API changes need to change at least minor version.
unless if you increment minor version every time you release.

Reply:
I bump minor version each release to stable (it's still TM:PE v11) -> 11.6 is still under testing on **Test** and I didn't push 11.6 tag in case of bugs